src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx

```diff
@@ -215,20 +258,33 @@ const displayPatientInfo = (
                     <PatientQuestionnaireResponses patientID={patient.id} />
 
                     <Table
-                        className="patient-table"
                         key={'SchemaTable'}
-                        rowKey={(record) => record.id}
-                        columns={columns}
+                        className="patient-table"
                         dataSource={dataSource}
-                        size="small"
-                        pagination={{ pageSize: 10 }}
+                        columns={columns}
+                        rowKey={(record) => record.id}
+                        rowClassName={(record, index) =>
+                            index % 2 === 0
+                                ? 'table-row-light'
+                                : 'table-row-dark'
+                        }
                         onRow={(record) => {
                             return {
                                 onClick: () => {
                                     handleClick(record);
                                 },
                             };
                         }}
+                        pagination={{ pageSize: 10 }}
+                        size="small"
+                        bordered
+                        showSorterTooltip={true}
+                        locale={{
+                            sortTitle: 'test',
+                            cancelSort: 'Avbryt sortering',
```

Comment:
I think "Nullstill sortering" or just "Nullstill" gives a better description of this function. What do you think?

Reply:
Agreed.